crates/cdk-sql-common/src/mint/migrations.rs

```diff
@@ -1,6 +1,7 @@
 /// @generated
 /// Auto-generated by build.rs
 pub static MIGRATIONS: &[(&str, &str, &str)] = &[
+    ("", "20250812132015_drop_melt_request.sql", include_str!(r#"./migrations/20250812132015_drop_melt_request.sql"#)),
```

Comment:
Is this right? I think it should only be sqlite and why is it included in this pr?

Reply:
Looks like https://github.com/cashubtc/cdk/pull/953/files was wrong